integrations/ScriptLoader.js

```diff
@@ -6,9 +6,13 @@ const ScriptLoader = (id, src) => {
   js.src = src;
   js.async = true;
   js.type = "text/javascript";
+  if(id === "TVSquared-integration"){
+    js.defer = true;
+  }
```

Comment:
currently we load all js as `async` only. remove the `defer` attribute now. we have the task of making all the scripts with `defer` in pipeline. Will do the change as part of that task.

Reply:
ok